build.sbt

```diff
@@ -303,10 +303,49 @@ lazy val dagShared = (project in file("modules/dag-shared"))
     )
   )
 
+lazy val circeVersion         = "0.11.0"
+lazy val finagleVersion       = "6.45.0"
+lazy val finchVersion         = "0.15.1"
+lazy val scalaTestVersion     = "3.0.0"
+
+lazy val rosetta = (project in file("modules/rosetta"))
+  .enablePlugins(JavaAppPackaging)
+  .dependsOn(shared % "compile->compile;test->test",
+    testShared % Test, keytool, sdk, wallet, dagShared)
+  .settings(
+    name := "tessellation-rosetta",
+    Defaults.itSettings,
+    scalafixCommonSettings,
+    commonSettings,
+    commonTestSettings,
+    makeBatScripts := Seq(),
+    libraryDependencies := Seq(
+      CompilerPlugin.kindProjector,
+      CompilerPlugin.betterMonadicFor,
+      CompilerPlugin.semanticDB,
+      Libraries.logback % Runtime,
+      Libraries.circeCore,
+      Libraries.circeGeneric,
+      Libraries.circeParser,
+      Libraries.circeRefined,
+      Libraries.circeShapes,
+      Libraries.http4sDsl,
+      Libraries.http4sServer,
+      Libraries.http4sClient,
+      Libraries.http4sCirce,
+      Libraries.http4sJwtAuth,
+      Libraries.doobieCore,
+      Libraries.doobieHikari,
+      Libraries.doobieQuill,
```

Comment:
I will be using `Quill` for serialization and de-serialization when I write to and read from the database. Can you recommend a replacement.

Reply:
You can remove these dependencies here because this module doesn't require it. On later PRs we can discuss changes to the data store methods if we want to change serialization

Reply:
Dependencies removed.